app/Module.scala

```diff
@@ -60,6 +60,17 @@ class Module(environment: Environment,
     ElasticSettings(localMode, homeDir, preDelete, connectionString, isCluster, clusterName, numShards)
   }
 
+  @Provides
+  @Singleton
+  def provideRdsQueryConfig(configHelper: ConfigHelper): RdsQueryConfig = {
+    val queryTimeoutMillis =
+      configHelper.getConfigString("address-lookup-rds.query-timeout-ms").map(_.toInt).getOrElse(10000)
```

Comment:
Should we parameterise these and put them in config to make it easy to tweak in the future?

Reply:
actually looks like they are in app conf, just not used

Reply:
oh - what makes you say that the application.conf values aren't being used?